pyproject.toml

```diff
@@ -76,10 +76,11 @@ warn_unused_configs = true
 warn_unused_ignores = true
 warn_incomplete_stub = true
 show_error_codes = true
+exclude = 'openff/interchange/_tests'
 
 [[tool.mypy.overrides]]
 module = [
-  "pandas",
```

Comment:
(not blocking) Just double checking that this PR intends to remove pandas from this list.